features-support/step_definitions/features.js

```diff
@@ -106,12 +96,19 @@ module.exports = function () {
       });
   });
 
-  this.Then(/^the scenarios will be visible\.$/, function (callback) {
+  this.Then(/^the scenarios will be visible\.?$/, function (callback) {
     should.equal(this.statusCode, 200, "Bad HTTP status code.");
-    should.equal(/feature/i.test(this.body),
+    should.equal(/feature:/i.test(this.body),
       true,
       "The returned document body does not contain the word 'feature'");
     callback();
   });
 
+  this.Given(/^a URL representing a remote Git repo\.?$/, function (callback) {
+    this.repoUrl = "https://github.com/oss-specs/specs";
```

Comment:
maybe accept specs url as a step argument, we'll be able to swap it out easily. Or leave it and change it when needed, up to you